pkg/services/sqlstore/annotation.go

```diff
@@ -127,6 +127,7 @@ func (r *SQLAnnotationRepo) Find(query *annotations.ItemQuery) ([]*annotations.I
 			annotation.epoch as time,
 			annotation.epoch_end as time_end,
 			annotation.dashboard_id,
+			dashboard.uid as dashboard_uid,
```

Comment:
This is a small detail, @cunyat but it occurred to me from writing and reading lots of SQL in the past:

It looks like the previous SQL authors here have attempted to group all of the columns in this file by the table name.

If so, it might be a slight consistency improvement to move this `dashboard.uid` line after the `annotation.updated` line.